scripts/tests-ui-integration.sh

```diff
@@ -3,46 +3,29 @@
 PUBLISHED_FUEL_PACKAGE_NAME="${PUBLISHED_FUEL_PACKAGE_NAME-"fuels"}"
 PUBLISHED_NPM_TAG="${PUBLISHED_NPM_TAG-"next"}"
 
-# Versions
-FUEL_CORE_VERSION=$(cat ./internal/fuel-core/VERSION)
-FORC_VERSION=$(cat ./internal/forc/VERSION)
-TOOLCHAIN="CI"
-
 # Project
-ROOT_DIR=$(pwd)
-PLAYWRIGHT_DIR="$ROOT_DIR"
-PROJECT_DIR="$ROOT_DIR/test-project"
```

Comment:
I moved the `PROJECT_DIR` to a temporary directory generated via
```sh
PROJECT_DIR="$(mktemp -d)/integration-test-project"
```
because then we've got a clean slate and don't need to have the workarounds below in the `"3. Intialise the project"` step related to installation because the test project in no longer living under the same workspace as our repository.